packages/kbn-config-schema/src/types/string_type.test.ts

```diff
@@ -71,14 +85,13 @@ describe('#hostname', () => {
     expect(hostNameSchema.validate('www.example.com')).toBe('www.example.com');
     expect(hostNameSchema.validate('3domain.local')).toBe('3domain.local');
     expect(hostNameSchema.validate('hostname')).toBe('hostname');
-    expect(hostNameSchema.validate('2387628')).toBe('2387628');
```

Comment:
`joi` hostname validation was improved

DOMAIN_INVALID_TLDS_CHARS (last segment must start with alphanum) - https://github.com/sideway/address/blob/master/lib/domain.js#L88

Reply:
Looking at the changelog for joi 17.0.0, it appears that they also added support for non-ascii chars in hostnames: https://github.com/sideway/joi/issues/2163 😄